test/client/bulk_test.rb

```diff
@@ -300,30 +308,14 @@
     assert_bulk_index(items[0])
 
     assert_equal "foo", items[0]["index"]["_id"]
-  end
-
-  it "supports symbol and string parameters" do
-    response = @index.bulk do |b|
-      doc1 = { author: "pea53", message: "a tweet about foo" }
-      b.index doc1, { id: "foo", type: "tweet" }
-
-      doc2 = { author: "pea53", message: "a tweet about bar" }
-      b.index doc2, { "id" => "bar", "type" => "tweet" }
-    end
-
-    assert_kind_of Integer, response["took"]
+    assert_equal "Book 1", @index.docs("book").get(id: "foo")["_source"]["title"]
 
-    items = response["items"]
-
-    assert_bulk_index(items[0])
-    assert_bulk_index(items[1])
-
-    assert_equal "a tweet about foo", @index.docs("tweet").get(id: "foo")["_source"]["message"]
-    assert_equal "a tweet about bar", @index.docs("tweet").get(id: "bar")["_source"]["message"]
+    assert_equal "bar", items[1]["index"]["_id"]
+    assert_equal "Book 2", @index.docs("book").get(id: "bar")["_source"]["title"]
   end
 
   it "doesn't override parameters from the document" do
```

Comment:
```suggestion
  it "doesn't override parameters with properties from the document" do
```
A minor wording suggestion to an existing test name to make it clearer. First time I read it, I thought "parameters from the document" meant properties that came from the document.